src/CMakeLists.txt

```diff
@@ -128,7 +128,11 @@ ELSE()
   FOREACH(DIR ${KK_INCLUDE_DIRS})
     TARGET_INCLUDE_DIRECTORIES(kokkoskernels PUBLIC $<BUILD_INTERFACE:${DIR}>)
   ENDFOREACH()
-  TARGET_INCLUDE_DIRECTORIES(kokkoskernels PUBLIC $<INSTALL_INTERFACE:include>)
+  IF(KOKKOSKERNELS_HAS_PARENT)
+    TARGET_INCLUDE_DIRECTORIES(kokkoskernels PUBLIC $<INSTALL_INTERFACE:include/kokkos-kernels>)
```

Comment:
You would again still want to the add this include directory for installation. The INSTALL_INTERFACE generator expression only adds this to the install, not the in-tree build.